### PR TITLE
Fix css styling not applying for eula container

### PR DESCRIPTION
**Pull request by JMarshall841**

### CODE REVIEW

**Review by dwinn — APPROVED** [review]
Looks good to me!

File: lib.php [hunk 1]
```diff
@@ -466,15 +466,15 @@ public function print_disclosure($cmid) {
             return html_writer::tag(
                 'div',
                 get_string('eulaalreadyaccepted', 'plagiarism_turnitinsim'),
-                array('class' => 'eulacontainer', 'id' => 'eulacontainer')
+                array('class' => 'turnitinsim_eulacontainer', 'id' => 'turnitinsim_eulacontainer')
```
Comment (grijeshsaini) [inline lib.php:469]: Do we need to change the id as well?

File: amd/src/eula_response.js [hunk 2]
```diff
@@ -44,15 +44,15 @@ define(['jquery', 'core/str'], function($, str) {
                     data: {action: "accept_eula", sesskey: M.cfg.sesskey},
                     success: function() {
                         str.get_string('eulaaccepted', 'plagiarism_turnitinsim').done(function(text) {
-                            $('.eulacontainer').hide().html(text).fadeIn();
+                            $('.turnitinsim_eulacontainer').hide().html(text).fadeIn();
```
Comment (grijeshsaini) [inline amd/src/eula_response.js:47]: I generally prefer finding element by id but I am fine here as it was searching with a class name before.

**Review by grijeshsaini — APPROVED** [review]

File: lib.php (hunk 1 above)
Comment (JMarshall841) [reply lib.php:469]: @dwinn Do you know if the Moodle standard for prefixing html containers with the plugin name applies to both class and id?

**Pull request merged**